x-pack/plugins/actions/server/routes/connector/get_all/transforms/transform_connectors_response/v1.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FindConnectorResult } from '../../../../../application/connector/types';
+import { ConnectorResponse } from '../../../../../../common/routes/connector/response';
+import { RewriteResponseCase } from '../../../../../../common/rewrite_request_case';
+
+export const transformGetAllConnectorsResponse: RewriteResponseCase<FindConnectorResult[]> = (
+  results
+): ConnectorResponse[] => {
```

Comment:
Let's use the versioned type here since this is also a versioned file (`ConnectorResponseV1`). It let's us know to increment this file when the underlying schema has changed